autoload/vim_ai.vim

```diff
@@ -1,4 +1,13 @@
 call vim_ai_config#load()
+if &compatible || exists('g:loaded_vim_ai')
+    finish
+endif
+
+function! s:vim_ai()
+    
+endfunction
+
+let g:loaded_vim_ai = 1
```

Comment:
```suggestion
```

Reply:
Thank you, the last quick fix unintentionally introduced boilerplate code; now removed.